sonarqube-scanner/src/python/badfortune.py

```diff
@@ -87,6 +87,6 @@ def get(filename):
 if __name__ == '__main__':
     import sys
     if len(sys.argv) == 1:
-        print 'Usage: fortune.py <filename>'
+        print  'Usage: fortune.py <filename>' 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace print statement by built-in function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://18.236.171.46:9000/coding_rules#rule_key=python%3APrintStatementUsage)